### PR TITLE
Updated the readme with instructions to help devs pass the pep8 tests.

### PR DESCRIPTION
**Pull request by ragv**
Updated readme with Installation and usage instructions for `autopep8`.

### CODE REVIEW

**Comment by ghost** [comment]
@debugger22 Check this out !

**Comment by debugger22** [comment]
I'm a bit confused here. Should we list it as a dependency or should we distribute source or binary? @salil93 @jayrambhia see this.

**Comment by ghost** [comment]
Same confusion here ... !! but i feel pep8 test ( points out bad indentation ) => autopep8 dependency ( a tool to correct those bad indentation automatically ) ? 
also the binary looks cooler and cleaner but will it not be specific to a system ? i.e It wont work with windows I think ...
Other alternative is distributing the source !

**Comment by rajat974** [comment]
autopep8 is always not in sync with pep8. They are different projects. Adding autopep8 does not ensure the pep8 indentation. We can do it manually. because there are still many errors which are not handled by autopep8.

**Comment by salilkapur** [comment]
I dont think autopep8 should be a dependency.

**Comment by ghost** [comment]
@salil93 How about distributing the source code of the same and installing it parallelly along with binpy setup ? or should we just leave it for the devs to install it somehow ?

**Comment by rajat974** [comment]
we can add that in README or  installation instruction with all the necessary commands for autopep8ing and checking pep8 errors

**Comment by ghost** [comment]
@rajat974 Thats looks fine ! I'll do that and push the commit in some time!

**Comment by ghost** [comment]
@salil93 Please review it now :)

File: README.md [hunk 1]
```diff
@@ -163,5 +165,20 @@ How To Contribute
...
+ 
+ - After installation is complete. Make the necessary changes and commit your changes. After Committing your changes, `cd` to the BinPy root directory and issue the following command
+
+   `autopep8 -r -i -a -a -v .`
+   
+   To learn more about the `autopep8` tool visit [here](https://www.google.co.in/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=0CCkQFjAA&url=https%3A%2F%2Fpypi.python.org%2Fpypi%2Fautopep8%2F&ei=SjFIU7jkIcWKrQfE5oDgBQ&usg=AFQjCNGP0o38e1Ia6S7_TfsDIJrvgdGAug&sig2=Yp4VZe9UepdYtoCF_mcBFg).
+
+ - Then migrate to the `BinPy/BinPy/tests` directory and run tests by issuing the command `nosetests` [ For Python 2 tests] and `nosetests3` [ For Python 3 tests ].
+
+If all the tests pass successfully push your repo to the origin/branch and send us a Pull Request. We'll be happy to review the same and merge it with our codebase.
```
Comment (rajat974) [inline README.md:182]: This probably is pep8 inconsistent :D, shorten the line length :D

Comment (rajat974) [inline README.md:182]: Also mention about cross checking with `pep8 ./ --ignore=E501`

Comment (rajat974) [inline README.md:182]: Also add somewhere autopep8 docs link

Comment (ragv) [inline README.md:182]: 1. lol :D 
2. Added!
3. check line 176 ;)

Comment (rajat974) [inline README.md:182]: how nosetest will check until a test is written?? check once again. and add `pep8 ./ --ignore=E501`

**Comment by debugger22** [comment]
Looks good to me now :+1:

**Comment by debugger22** [comment]
@rvraghav93 Just squash the commits into one meaningful commit.

**Comment by debugger22** [comment]
Thanks!

**Pull request merged**